internal/binutils/binutils_test.go

```diff
@@ -188,15 +190,51 @@ func skipUnlessDarwinAmd64(t *testing.T) {
 	}
 }
 
+// ensureObjdump checks if a supported version of objdump is availabe. Keep the test simple
+// and check for LLVM objdump v11.0+ or a GNU Binutils objdump.
+func ensureObjdump() bool {
+	cmdOut, _ := exec.Command("objdump", "--version").Output()
+	// Keep this regexp in sync with the one used in binutils.go
+	//	re := regexp.MustCompile(`LLVM\sversion (([0-9]*)\.([0-9]*)\.([0-9]*)).*\s`)
+	fields := objdumpLLVMVerRE.FindStringSubmatch(string(cmdOut))
+	if len(fields) == 5 {
+		if ver, _ := strconv.Atoi(fields[2]); ver >= 11.0 {
+			return true
+		}
+	}
+
+	cmdOut, _ = exec.Command("objdump", "--version").Output()
+	if strings.Contains(string(cmdOut), "GNU") && strings.Contains(string(cmdOut), "Binutils") {
+		return true
+	}
+
+	cmdOut, _ = exec.Command("gobjdump", "--version").Output()
+	if strings.Contains(string(cmdOut), "GNU") && strings.Contains(string(cmdOut), "Binutils") {
+		return true
+	}
+	return false
+}
+
 func testDisasm(t *testing.T, intelSyntax bool) {
+	// The check is only performed on MacOS. Linux usually has a supported
+	// version of objdump available.
+	if runtime.GOOS == "darwin" && !ensureObjdump() {
+		t.Skip("cannot find usable objdump")
```

Comment:
nit: Cannot

(use sentence case in t.{Log*,Skip} methods)

Reply:
done.